index.html

```diff
@@ -50,5 +50,162 @@ <h1 class="head-h1">Hey there.<br>I'm Edu.<span><h2>I'm a software developer</h2
         </ul>
       </div>
     </section>
+    <section class="works-container" id="works">
+      <div class="works-header-container header-area">
+        <div class="works-header">
+          <h1>My recent works</h1>
+          <div class="line"></div>
+        </div>
+        <div class="works-header-multipost">
+          <img class="img-placeholder" src="./img/placeholders/Img Placeholder.png" alt="placeholder-image">
+          <div class="works-stories">
+            <h1>Multi-Post Stories</h1>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <div class="languages-box">
+              <ul class="languages-list">
+                <li>CSS</li>
+                <li>HTML</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+                <a tabindex="0" href="#" class="btn-action btn-area btn">See Project</a>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="project-box left" id="project-1">
+        <div class="project-content">
+          <div class="project-text">
+            <h1>Profesional Art<br>Printing Data</h1>
```

Comment:
- [ ] The html standard is that you should use only one `h1` tag in your whole application. you can visit this [link](https://www.reliablesoft.net/h1-tag/) and watch this [video](https://www.youtube.com/watch?v=rH-4w_0F1vM) to learn more about this. you can use any `hx` tag such as `h1, h2` etc.